node/util/node_tt.C

```diff
@@ -306,7 +306,7 @@ local char * alt_construct_merger_label(node * ni, node * nj)
     // Decode the names and combine them to make the new name.
     // Not very efficiently written, but... (Steve, 12/98)
 
-    static char new_name[1024];
+    static char new_name[1040];
```

Comment:
why 1040? seems a bit random to increase by 16...

Reply:
The compiler does the math and tells you how much space is needed. I think in this case it's 1024 for the string, 11 for a 32-bit int plus sign, 2 more for the `<` and `>`, and finally the terminating NULL, which adds up to 1040.

Of course in practice the string will probably be shorter than 1024 characters, but unless we switch to dynamically allocated strings everywhere, this is what we get.